app/views/admin/neighbourhoods/index.html.erb

```diff
@@ -25,3 +26,4 @@ var columns = [
 <p><%= link_to safe_neighbourhood_name(root), admin_neighbourhood_path(root) %></p>
 <% end %>
 
+<h5>Latest Release Date: <%= Neighbourhood::LATEST_RELEASE_DATE %></h5>
```

Comment:
nit pick / maybe non-issue but maybe this could be explicit about it being ONS dataset from that date. just so there's no ambiguity about what has been released.

Reply:
have changed to "ONS Latest Release Date Loaded: `<%= Neighbourhood::LATEST_RELEASE_DATE %>`"